src/blockchain_utilities/README.md

```diff
@@ -100,21 +83,11 @@ BerkeleyDB flags (takes one):
 
 ```
 ## Examples:
-$ blockchain_import --database lmdb#fastest
-$ blockchain_import --database berkeley#fastest
+$ monero-blockchain-import --database lmdb#fastest
+$ monero-blockchain-import --database berkeley#fastest
```

Comment:
I think we can remove examples for bdb, I don't think it'd actually work anymore